roles/fio_distributed/templates/server_vm.yml.j2

```diff
@@ -13,52 +13,53 @@ spec:
       cores: {{ workload_args.vm_cores | default(1) }}
     devices:
       disks:
-        - name: registrydisk
-        - name: cloudinitdisk
+        - disk:
+            bus: virtio
```

Comment:
I agree this should be default but would it be useful to allow user to override this?  Does KVM or CNV support other forms of block device driver and would we ever need to test them?

Reply:
The other bus option is sata which can have very poor performance, however it is the default for usability reasons (for ex. if the guest OS didn't have virtio drivers installed). I can add it as an option, I just don't want to encourage it's usage due to the performance implications :)

Reply:
I agree completely, that's why virtio should be the default here, but it is possible to allow the user to set the driver and this might be useful precisely so we can demonstrate to people how bad the alternatives are, for example.   THere is more than 1 alternative if I remember right.

Reply:
Ah yeah I forgot about scsi: http://kubevirt.io/api-reference/master/definitions.html#_v1_disktarget  even more of a reason to add an option w/ default set to virtio